pytype/tools/merge_pyi/merge_pyi.py

```diff
@@ -626,6 +626,12 @@ def insert_comment_annotation(self, src_sig, pyi_sig):
       return []
 
   def can_annotate(self, src_sig):
+    if src_sig.short_name.startswith('__') and \
```

Comment:
We follow the Google Python style guide, which forbids backslash line continuations (http://google.github.io/styleguide/pyguide.html#32-line-length). Could you find another way to break up this expression?

Reply:
Yeah, used parentheses for an implicit line joining.